services/credentials.go

```diff
@@ -86,6 +85,23 @@ func AuthValidityWindow(ot credentials.OperatorType) time.Duration {
 	return quota.authValidityWindow
 }
 
+func GetQuotaJSON(ot credentials.OperatorType) (*json.RawMessage, error) {
+	quotaData := map[string]interface{}{
+		"count":              uint(credsQuota(ot)),
+		"window":             credsQuotaWindow(ot),
+		"authValidityWindow": AuthValidityWindow(ot),
+	}
+
+	jsonData, err := json.Marshal(quotaData)
+	if err != nil {
+		return nil, err
+	}
+
+	var quotaJson json.RawMessage = jsonData
```

Comment:
just change the signature to return `json.RawMessage` instead of `*json.RawMessage` and you won't have to take the address of the byte slice. `nil` is a valid json.RawMessage since the underlying type is a byteslice which is nillable.

Ie, you can relace everything after L3 with `return json.Marshal(quotaData)`

Reply:
This is admittedly a bit beyond my understanding as far as when to return a pointer vs a value, but the change was easy enough. :)